.github/workflows/lint_python.yml

```
@@ -0,0 +1,28 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v3
+      - run: sudo apt upgrade -y && sudo apt-get install curl ffmpeg libcurl4-openssl-dev libssl-dev motion ssh v4l-utils -y
```

```suggestion
       - run: |
           sudo apt-get -q update
           sudo apt-get -qq upgrade
           sudo apt-get -qq install curl ffmpeg libcurl4-openssl-dev libssl-dev motion ssh v4l-utils
```
- `apt-get` is better suited for non-interactive/script executions.
- `-qq` implies `-y` and produces less processing output which isn't nice to read in GitHub actions logs.

---

`apt-get -q update` is VERY expensive (90+ seconds) and is probably not required for our tests.